schema-registry/rest-service/src/main/java/com/hortonworks/registries/schemaregistry/webservice/ConfluentSchemaRegistryCompatibleResource.java

```diff
@@ -284,10 +286,10 @@ public Response lookupSubjectVersion(@ApiParam(value = "Schema subject", require
     @POST
     @Path("/subjects/{subject}/versions")
     @ApiOperation(value = "Register a new version of the schema",
-            notes = "Registers the given schema version to schema with name if the given schemaText is not registered as a version for this schema, " +
+            notes = "Registers the given schema version to schema with subject if the given schemaText is not registered as a version for this schema, " +
                     "and returns respective unique id." +
                     "In case of incompatible schema errors, it throws error message like 'Unable to read schema: <> using schema <>' ",
-            response = Id.class, tags = OPERATION_GROUP_CONFLUENT_SR)
+            response = VersionId.class, tags = OPERATION_GROUP_CONFLUENT_SR)
```

Comment:
this should be Id.class and should be the global id returned.

see api operation note "and returns respective unique id."

Reply:
My bad